src/java/org/apache/cassandra/cql3/functions/FunctionCall.java

```diff
@@ -150,7 +153,11 @@ public static Raw newCast(Term.Raw raw, CQL3Type type)
 
         public Term prepare(String keyspace, ColumnSpecification receiver) throws InvalidRequestException
         {
-            Function fun = FunctionResolver.get(keyspace, name, terms, receiver.ksName, receiver.cfName, receiver.type);
+            KeyspaceMetadata ksm = ClusterMetadata.current().schema.getKeyspaces().getNullable(name.hasKeyspace() ? name.keyspace : keyspace);
```

Comment:
nit: this duplicated code could be pulled into a method on `ClusterMetadata`, `DistributedSchema` or `UserFunctions`.